doc/tutorials/tarball_installation.rst

```diff
@@ -0,0 +1,249 @@
+.. _tar_installation:
+
+ACRN Installation via Tar Files
+####################################
+
+Project ACRN offers two ways to install ACRN on target systems, either via
+Debian packages or tar files. This document covers the tar file method. For
+information about the Debian method, see :ref:`debian_packaging`.
+
+Tar files provide a way to package ACRN configurations for Linux
+target systems running non-Debian operating systems. You generate the tar
+files on your development computer, then copy the files onto your target system
+and reboot the system with ACRN up and running.
+
+This document shows examples of commands used to build and install the tar
+files. You might need to modify the commands for your environment.
+
+Prerequisites
+*************
+
+* A development computer configured according to the :ref:`gsg` (for example, it
+  has an Ubuntu OS, ACRN dependencies, and ACRN hypervisor and kernel source
+  code).
+
+Build the Tar Files
+***************************
+
+#. On your development computer, navigate to the ``acrn-hypervisor`` source code
+   and build the ACRN hypervisor tar files. Be sure to replace
+   ``<board.xml>`` and ``<scenario.xml>`` with the paths to your board
+   configuration file and scenario configuration file.
+
+   .. code-block:: bash
+
+      cd ~/acrn-work/acrn-hypervisor
+      make -j $(nproc) BOARD=<board.xml> SCENARIO=<scenario.xml>
+      make targz-pkg
+
+#. Navigate to the ``acrn-kernel`` source code and build the ACRN kernel tar
+   files for the Service VM:
+
+   .. code-block:: bash
+
+      cd ~/acrn-work/acrn-kernel
+      cp kernel_config_service_vm .config
+      make olddefconfig
+      make -j $(nproc) targz-pkg
+
+Install and Run ACRN
+**************************
+
+In the following steps, you will install the tar files, install the serial
+configuration tool, configure GRUB, and run ACRN on the target system.
+
+#. Copy all the necessary files generated on the development computer to the
+   target system by USB disk as follows. Modify the file names in the following
+   commands to match your files.
+
+   a. Insert the USB disk into the development computer and run these commands:
+
+      .. code-block:: bash
+
+         disk="/media/$USER/"$(ls /media/$USER)
+         cp linux-5.10.65-acrn-service-vm-x86.tar.gz "$disk"/
+         cp ~/acrn-work/acpica-unix-20210105/generate/unix/bin/iasl "$disk"/
+         cp ~/acrn-work/acrn-hypervisor/build/acrn-2.7-unstable.tar.gz "$disk"/
+         sync && sudo umount "$disk"/
+
+      .. note:: The :file:`serial.conf` is only generated if non-standard
```

Comment:
This note can now be moved further down when we install the `setserial` tool

Reply:
@gvancuts I didn't make this change, because your other comment says that mentions of serial.conf file can all go away completely. Please check.

Reply:
You removed the note completely, didn't you?

The way I understand this you only really need to install the `setserial` tool **if** you have a non-standard serial port. In that case you should install `setserial` that will use the `serial.conf` that we generated and installed. I'm thinking the easiest is to ask the user to install `setserial` regardless, I don't know that there would be any side-effect even in the case you only have standard serial ports configured. @NanlinXie , can someone confirm?

Reply:
@gvancuts Yes, I deleted the note.

Reply:
Sounds good, thanks @amyreye !

@NanlinXie , one more question to the engineering team, is there any other action required to get the non-standard serial port configured? Or is installing the `setserial` package and the custom `serial.conf` is sufficient? I believe it is but I do not have first hand experience with this.

Reply:
@wuxyintel , could you help to reply @gvancuts's question?

Reply:
@gvancuts @NanlinXie for acrn v2.7, non-standard ports are emulated by hypervisor only when user config them in scenario file manually, I think these non-standard ports in scenario file will be configured in scenario file by default in acrn v2.8.
If non-standard ports are emulated by hypervisor, serial.conf will be generated during hypervisor build-time. Installing the setserial package and the custom serial.conf in service VM is sufficient. Thanks!